src/groups/rotation_translation_action.jl

```diff
@@ -0,0 +1,310 @@
+@doc raw"""
+    RotationTranslationAction(
+        M::AbstractManifold,
+        SOn::SpecialEuclidean,
+        AD::ActionDirection = LeftForwardAction(),
+    )
+
+Space of actions of the [`SpecialEuclidean`](@ref) group $\mathrm{SE}(n)$ on a
+Euclidean-like manifold `M` of dimension `n`.
+
+Left forward action corresponds to active transformations while right forward actions
+can be identified with passive transformations for a particular choice of a basis.
+"""
+struct RotationTranslationAction{
+    TAD<:ActionDirection,
+    TM<:AbstractManifold,
+    TSE<:SpecialEuclidean,
+} <: AbstractGroupAction{TAD}
+    manifold::TM
+    SEn::TSE
+end
+
```

Comment:
Could we document a few of the functions below? Mainly for a new user to more understand how the functions are related to actions happening in theory.

Reply:
This is still WPI, I will do that later.

Reply:
I've added some documentation here.